src/ci/ci-rerun.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+const { default: chalk } = require('chalk');
+const program = require('commander');
+const got = require('got');
+
+const { fatal } = require('../utils/logging');
+
+program
+  .description('Rerun CI workflows')
+  .argument('[workflow]', 'The ID of the workflow to rerun')
+  .option('-f, --from-failed', 'Rerun workflow from failed', false)
```

Comment:
Can we make this default to true instead? I think the majority of calls should use it